### ISSUE TITLE
Canvas appearing out of place

### ISSUE THREAD

**Issue by rgins16** [danialfarid/ng-file-upload#1707]
When I run your fiddle here:
http://jsfiddle.net/danialfarid/xxo3sk41/590/
Everything works as expected, including being able to crop a photo.

I copy and pasted that fiddle to my own app, and this is the result.
![image](https://cloud.githubusercontent.com/assets/16677164/18561720/070a4d76-7b4f-11e6-968d-e99106575d64.png)

The canvas is appearing out of place, and is not the right size.

![image](https://cloud.githubusercontent.com/assets/16677164/18561841/830a1c30-7b4f-11e6-8eaa-9e9f0a90fc28.png)

**Comment by andrewhyeung** [comment]
Yeah I am getting a similar issue.. 
any luck? 
<img width="649" alt="screen shot 2016-09-15 at 10 12 44 pm" src="https://cloud.githubusercontent.com/assets/16911354/18575955/974517bc-7b91-11e6-9a5c-b98a5149a944.png">

**Comment by rgins16** [comment]
unfortunatey, no

**Comment by rgins16** [comment]
So I fixed the issue I was having. I was including the wrong css file for ng-img-crop.

The confusion was due to there being an unminified folder, minified folder, source folder, etc...
The author's GitHub page gave no clear instructions on which folder to include the files from.

This is what worked for me.

CSS file: link(rel='stylesheet' type='text/css' href='/node_modules/ngImgCrop/compile/unminified/ng-img-crop.css')
JS file: script(src='/node_modules/ngImgCrop/compile/unminified/ng-img-crop.js')
The above is in Jade
